From 9eda36c8477feb2161e0a99d2618616d3557b37d Mon Sep 17 00:00:00 2001
From: Christian Marangi <ansuelsmth@gmail.com>
Date: Sun, 6 Jul 2025 11:08:37 +0200
Subject: [PATCH 1/3] dt-bindings: clock: Define ID for Switch MDC clock for
 EN7581

Airoha EN7581 provide a dedicated clock for the internal Switch MDC.
Define an ID to expose it to the system.

Signed-off-by: Christian Marangi <ansuelsmth@gmail.com>
---
 include/dt-bindings/clock/en7523-clk.h | 2 ++
 1 file changed, 2 insertions(+)

--- a/include/dt-bindings/clock/en7523-clk.h
+++ b/include/dt-bindings/clock/en7523-clk.h
@@ -17,4 +17,6 @@
 #define AN7583_CLK_MDIO0	9
 #define AN7583_CLK_MDIO1	10
 
+#define EN7581_CLK_SWITCH_MDC	9
+
 #endif /* _DT_BINDINGS_CLOCK_AIROHA_EN7523_H_ */
